sys/include/net/sock/dtls.h

```diff
@@ -794,7 +794,7 @@ static inline int sock_dtls_session_create(sock_dtls_t *sock,
         return res;
     }
 
-    return sock_dtls_recv(sock, remote, buf, sizeof(buf), timeout);
+    return sock_dtls_recv(sock, remote, buf, sizeof(buf), timeout) == -SOCK_DTLS_HANDSHAKE;
```

Comment:
Now it will return 1 on success :wink: 

```suggestion
    res = sock_dtls_recv(sock, remote, buf, sizeof(buf), timeout);
    return res == -SOCK_DTLS_HANDSHAKE ? 0 : res;
```

Reply:
*Cough* Thanks!